sdk/src/main/java/com/google/cloud/dataflow/sdk/io/CompressedSource.java

```diff
@@ -375,11 +384,34 @@ public T getCurrent() throws NoSuchElementException {
       return readerDelegate.getCurrent();
     }
 
+    @Override
+    public final long getParallelismConsumed() {
+      if (splittable) {
+        return readerDelegate.getParallelismConsumed();
+      } else if (done) {
+        return 1;
+      }
+      return 0;
```

Comment:
This code is wrong. Last return is unreachable. If !splittable, consumed is 0, not 1.